ingestion/src/metadata/ingestion/source/dashboard/metabase/client.py

```diff
@@ -146,22 +147,81 @@ def get_collections_list(self) -> List[MetabaseCollection]:
             logger.warning("Failed to fetch the collections list")
         return []
 
+    def get_charts_dict(self) -> Dict:
+        charts_dict = {}
+        try:
+            resp_charts = self.client.get("/card")
+            if resp_charts:
+                for chart_data in resp_charts:
+                    chart = MetabaseChart.model_validate(chart_data)
+                    charts_dict[chart.id] = chart
+            return charts_dict
+        except Exception:
+            logger.debug(traceback.format_exc())
+            logger.warning("Failed to fetch the cards")
+        return {}
+
+    def _create_default_dashboard_details(
+        self, orphan_charts_id: List
+    ) -> MetabaseDashboardDetails:
+        """
+        Returns:
+            MetabaseDashboardDetails object representing the default dashboard containing orphaned charts
+        """
+        dashboard_data = {
+            "description": "Contains charts not associated with any dashboard",
+            "name": "Default Dashboard",
+            "id": "-1",
+            "card_ids": orphan_charts_id,
+        }
+        return MetabaseDashboardDetails(**dashboard_data)
```

Comment:
same lets not create dict and then expand when we can just directly define the object